13.0/s2i/bin/assemble

```diff
@@ -171,7 +172,41 @@ function configure_proxy() {
   xml="$xml\
        </proxy>"
     sed -i "s|<!-- ### configured http proxy ### -->|$xml|" $HOME/.m2/settings.xml
+    return
   fi
+
+  # if http(s)_proxy is set, use that.
+  proxy=${HTTPS_PROXY:-${https_proxy:=${HTTP_PROXY:=$http_proxy}}}
+  noproxy=${NO_PROXY:-$no_proxy}
+  if [ -n $proxy ]; then
+    proxyhost=$(echo $proxy | cut -d : -f 1,2)
+    proxyport=$(echo $proxy | cut -d : -f 3)
+    if [[ `echo $proxy | grep -i https` ]]; then
```

Comment:
Should this be some kind of "begins with https" so it can't be fooled with http://myhttpsproxy?

Reply:
yes, will fix